### ISSUE TITLE
Possible misuse of aria-label

### ISSUE THREAD

**Issue by Gotterbild** [validator/validator#779]
I don't think it is a misuse. It helps screenreaders, which makes content accessible by disabled persons.

`<abbr title="United States" aria-label="United States">US</abbr>`

![image](https://user-images.githubusercontent.com/23502743/55408261-51de6580-5568-11e9-825e-87f96e08078c.png)

**Comment by sideshowbarker** [comment]
Thanks for taking time to report this.

The `aria-label` warning the checker is reporting  is based on that guidance in the following sources:

* https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/
* https://www.w3.org/TR/using-aria/#practical-support-aria-label-aria-labelledby-and-aria-describedby

Neither of those sources mentions anything about `aria-label` being used with `abbr`.

If you believe that `aria-label` should be allowed for `article`, please post a comment at https://github.com/w3c/html-aam/issues/160

**Comment by scottaohara** [comment]
related to my comment in #1090.  this should continue to be an error.  it is not consistently recognized by screen readers to use aria-label on an abbr, nor should it be.

**Comment by sideshowbarker** [comment]
See #1090
